extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2015 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CSS} from '../../../build/amp-apester-media-0.1.css';
+import {user} from '../../../src/log';
+import {loadPromise} from '../../../src/event-helper';
+import {setStyles} from '../../../src/style';
+import {listenFor} from '../../../src/iframe-helper';
+
+/**
+ * AMP Apester-media
+ */
+class AmpApesterMedia extends AMP.BaseElement {
+
+  // /** @override */
+  // isLayoutSupported() {
+  //   return true;// (layout == Layout.CONTAINER);
+  // }
+
+  /** @override */
+  viewportCallback() {
+    // TODO: if amp left the viewport, send iframe post message that it has been seen.
+  }
+
+  /** @override */
+  buildCallback() {
```

Comment:
I've added it, but it looks like isExperimentOn is not exported. 

TAG const is 'amp-apester-media'.